### PR TITLE
Ccarfi sept 22 product update

### PR DESCRIPTION
**Pull request by ccarfi**
september release notes blog post

### CODE REVIEW

File: unlock-protocol-com/blog/sept-22-update.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Partners and integrations
+- Launched on [Arbitrum](https://unlock-protocol.com/blog/arbitrum)
+- Launched on [Avalanche](https://unlock-protocol.com/blog/avalanche)
+
+## Checkout 
+- [Support for fiat recurring subscriptions](https://unlock-protocol.com/blog/unlock-fiat-subscription)
+- [Support for Apple Pay and Google Pay mobile payments](https://unlock-protocol.com/blog/support-for-google-and-apple-pay)
+- [Password-protected NFT purchases](https://unlock-protocol.com/blog/password-required-hook) 
+
+## Members page
```
Comment (kalidiagne) [inline unlock-protocol-com/blog/sept-22-update.md:21]: We can remove this one, is part of an old Update, also the new dashboard includes the new members page

File: unlock-protocol-com/blog/sept-22-update.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- [Enhanced Members Page filters](https://unlock-protocol.com/blog/new-filters-members-page) 
+
+## Dashboard
```
Comment (searchableguy) [inline unlock-protocol-com/blog/sept-22-update.md:24]: Added support for airdroping keys in bulk using a spreadsheet.

**Review by searchableguy — APPROVED** [review]

File: unlock-protocol-com/blog/sept-22-update.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- Launched on [Arbitrum](https://unlock-protocol.com/blog/arbitrum)
+- Launched on [Avalanche](https://unlock-protocol.com/blog/avalanche)
+
+## Checkout 
+- [Support for fiat recurring subscriptions](https://unlock-protocol.com/blog/unlock-fiat-subscription)
+- [Support for Apple Pay and Google Pay mobile payments](https://unlock-protocol.com/blog/support-for-google-and-apple-pay)
+- [Password-protected NFT purchases](https://unlock-protocol.com/blog/password-required-hook) 
+
+## Members page
+- [Enhanced Members Page filters](https://unlock-protocol.com/blog/new-filters-members-page)
```
Comment (kalidiagne) [inline unlock-protocol-com/blog/sept-22-update.md:22]:
```suggestion
```

Comment (kalidiagne) [reply unlock-protocol-com/blog/sept-22-update.md:22]: We can remove this one, is part of an old Update, also the new dashboard includes the new members page

Comment (ccarfi) [reply unlock-protocol-com/blog/sept-22-update.md:22]: thanks @kalidiagne — going to leave it in the list as it WAS one of the many things we shipped in september. we move so fast, some things change like that!

**Pull request merged**